tests/integration-tests/tests/common/utils.py

```diff
@@ -52,8 +52,8 @@
     "rocky8": {"name": "Rocky-8-EC2-Base-8.8*", "owners": ["792107900819"]},  # TODO add china and govcloud accounts
     "rhel8.9": {"name": "RHEL-8.9*_HVM-*", "owners": RHEL_OWNERS},
     "rocky8.9": {"name": "Rocky-8-EC2-Base-8.9*", "owners": ["792107900819"]},  # TODO add china and govcloud accounts
-    "rhel9": {"name": "RHEL-9.3*_HVM-*", "owners": RHEL_OWNERS},
-    "rocky9": {"name": "Rocky-9-EC2-Base-9.3*", "owners": ["792107900819"]},  # TODO add china and govcloud accounts
+    "rhel9": {"name": "RHEL-9.*_HVM*", "owners": RHEL_OWNERS},
```

Comment:
should we be pinning to 9.4 until fsx lustre supports 9.5?

Reply:
This test has Lustre installation disabled for RHEL/Rocky. So we can go head

Reply:
I'm not sure if this is what we want.  We aren't going to release for an image that doesn't support lustre, so whats the point of testing it?  I think we should enable lustre and stick with supported OSes